src/objects/aff/zcl_abapgit_aff_registry.clas.abap

```diff
@@ -46,21 +46,22 @@ CLASS ZCL_ABAPGIT_AFF_REGISTRY IMPLEMENTATION.
 
 
   METHOD initialize_registry_table.
-    register( 'BGQC' ).
     register( 'CDBO' ).
     register( 'CHKC' ).
     register( 'CHKO' ).
     register( 'CHKV' ).
     register( 'DRTY' ).
     register( 'DTEB' ).
-    register( 'DRAS' ).
     register( 'DSFD' ).
+    register( 'DSFI' ).
+    register( 'DRAS' ).
     register( 'EVTB' ).
     register( 'EEEC' ).
     register( 'GSMP' ).
     register( iv_obj_type     = 'INTF'
               iv_experimental = abap_true ).
     register( 'SMBC' ).
+    register( 'SWCR' ).
     register( 'NONT' ).
```

Comment:
only expect SWCR to be added here, no changes to other stuff

Reply:
reverted the changes already